### DISCUSSION TITLE
Request für neuen Sensor ELV-SH-CTH

### DISCUSSION THREAD

**Discussion by foks09** [SukramJ/aiohomematic#1921] [General]
Hallo zusammen,

ich hoffe das ich hier richtig bin. ELV hat einen neuen Homematic IP kompatiblen Temperatur/RLF Sensor als Bausatz (ELV-SH-CTH) und dieser wird nicht in HA angezeigt. In der Homematic IP App werden sie angezeigt. Werden noch mehr Infos zum Sensor benötigt?

**Comment by SukramJ** [comment]
Nein Danke nicht nötig. Such mal in der Geräteliste nach der Seriennummer oder Modell.
Ggf. hilft auch ein Neustart.

**Reply by foks09** [reply]
Die Sensoren werden auch nach einem Neustart weder in der Geräteliste noch über die Suche angezeigt.

**Reply by SukramJ** [reply]
Hat bei mir heute mit den frisch gelieferten ELV-SH-CTH ohne Neustart funktoniert.

**Reply by foks09** [reply]
Ich habe die Integration gerade gelöscht und neu installiert. Was auffällt ist, dass die beiden Sensoren weiterhin nicht erkannt werden und in der Übersicht zur Installation nicht angezeigt werden.

**Reply by SukramJ** [reply]
Mit den Mega information lässt sich natürlich viel anfangen!!!!
Bitte liefer die Informationen die in einem Issue Template  abgefragt werden.
-  Versionen, Logs, Diagnostics, ... 

Bitte schau selber nach.
